cmd/mock.go

```diff
@@ -17,17 +17,26 @@ limitations under the License.
 package cmd
 
 import (
+	"fmt"
 	"os"
 	"os/signal"
 	"syscall"
 
+	"github.com/hpcloud/tail"
```

Comment:
The last commit was 6 years ago. Please consider a well-maintained library, or you can even implement it yourself. Such as https://stackoverflow.com/questions/8270441/go-language-how-detect-file-changing

Reply:
got it.